### PR TITLE
Removed .state from filter from eng and fr

### PR DESCRIPTION
**Pull request by aspiiire**
I'am really sorry 😅 when I've made the change on the french language prettier on save have formatted everything and removed white spaces, let me know if I have to restore it to the previous version thanks.

### CODE REVIEW

File: website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx [hunk 1]
```diff
@@ -6,10 +6,10 @@ Avant de commencer, assurez-vous de lire la section sur [les providers](/docs/co
 
 ### Combiner des providers
 
-Nous avons vu précédemment comment créer un simple provider. Mais la réalité est la suivante, 
+Nous avons vu précédemment comment créer un simple provider. Mais la réalité est la suivante,
 dans de nombreuses situations, un provider pourra vouloir lire l'état d'un autre provider.
 
-Pour ce faire, nous pouvons utiliser l'objet [ref] passé au
```
Comment (GitGud31) [inline website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx:12]: You should keep that one space at the end of a line. I just break those lines (meaning, they're intentional) 'for myselfe' so I can read properly when I'm translating to french. But wait for @rrousselGit 's confirmation.

File: website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx [hunk 2]
```diff
@@ -57,31 +57,33 @@ class TodoList extends StateNotifier<List<Todo>> {
...
 
 - créer [StateProvider], qui présente la méthode de filtrage actuellement sélectionnée :
 
- ```dart
-  enum Filter {
-    none,
-    completed,
-    uncompleted,
-  }
+```dart
```
Comment (rrousselGit) [inline website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx:68]: The indentation should stay

File: website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx (hunk 1 above)
Comment (aspiiire) [reply website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx:12]: I am really sorry, going to fix it now!

File: website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx (hunk 2 above)
Comment (aspiiire) [reply website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx:68]: Ok thank you and sorry again! Going to fix it now God Willing!

**Comment by aspiiire** [comment]
Could you please check, sorry I'm still new to pull's on forks, if you see anything strange let me know thanks

**Comment by aspiiire** [comment]
@rrousselGit There are issues with this pull request? I see that 2 weeks have passed, I have done something wrong?

**Comment by rrousselGit** [comment]
Ah sorry, forgot about it. Thanks!

**Pull request merged**